src/main/java/com/sonymobile/jenkins/plugins/mq/mqnotifier/MQNotifierConfig.java

```diff
@@ -122,8 +127,9 @@ public void start() throws Exception {
      * Load configuration on invoke.
      */
     public MQNotifierConfig() {
-        this.enableNotifier = false;    // default value
-        this.persistentDelivery = true; // default value
+        this.enableNotifier = false;        // default value
+        this.persistentDelivery = true;     // default value
+        this.enableVerboseLogging = false;  // default value
```

Comment:
Defaulting to true would make it so that this pull request doesn't change anything to existing installations, which I feel is preferrable.

Reply:
Yes, sure